### PR TITLE
[DEV-1087] Fix wrong tabs grouping

### PR DESCRIPTION
**Pull request by tommaso1**
#### Motivation and Context

Tabs on markdown renderer are not rendered properly: if a tab has multiple markdown blocks as children only the first one is shown, and the other shifts one for title.

#### How Has This Been Tested?

There's an unit test updated to handle the case of multiple children. To manually test this you can open any guides page having tabs like(http://localhost:3000/pago-pa/guides/avviso-pagamento/3.2.0/allegato-2/specifiche-tecniche/intestazione).
Before test run `npm run compile`.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/gitbook-docs/src/__tests__/parseContent.test.ts [hunk 1]
```diff
@@ -503,10 +507,28 @@ describe('parseContent', () => {
...
+    Massimo 90 caratteri, spazi compresi.
+
+    {% hint style="info" %}
+    Puoi contare i caratteri con un [qualsiasi strumento online](https://www.charactercountonline.com/).
+    {% endhint %}
+    {% endtab %}
+    {% endtabs %}
+    `;
+
+    // This yould have only one child
```
Comment (marcobottaro) [inline packages/gitbook-docs/src/__tests__/parseContent.test.ts:529]: You **yould** fix this comment :wink:

**Comment by marcobottaro** [comment]
I think that the problem is not solved. I still see the hints from the 3rd tab appearing on the 4th
image

**Comment by tommaso1** [comment]
Now navigating to `pago-pa/guides/avviso-pagamento/3.2.0/allegato-2/specifiche-tecniche/intestazione` should display it correctly 

proof

**Review by marcobottaro — APPROVED** [review]
It works as expected after `npm run compile`

**Review by datalek — APPROVED** [review]

**Review by kin0992 — APPROVED** [review]

**Pull request merged**